helmfile/values/prometheus-blackbox-exporter-sc.yaml.gotmpl

```diff
@@ -13,10 +24,44 @@ config:
         preferred_ip_protocol: "ip4"
         tls_config:
           insecure_skip_verify: true
-
 serviceMonitor:
   enabled: true
   targets:
+    {{- if .Values.prometheusBlackboxExporter.targets.rook }}
+    - name: rook
+      url: rook-ceph-mgr.rook-ceph.svc.cluster.local:9283/health
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
+    {{- end }}
+    {{- if .Values.prometheusBlackboxExporter.targets.nginx }}
+    - name: nginx
+      url: ingress-nginx-controller-admission.ingress-nginx.svc.cluster.local:443
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_400
+    {{- end }}
+    {{- if .Values.prometheusBlackboxExporter.targets.fluentd }}
+    - name: fluentd
+      url: fluentd-forwarder.fluentd.svc.cluster.local:9880/fluentd.healthcheck?json={"ping","pong"}
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
+    {{- end }}
+    {{- if .Values.prometheusBlackboxExporter.targets.thanosQuery }}
+    - name: thanos
+      url: thanos-query-query.thanos.svc.cluster.local:9090
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
```

Comment:
Should we perhaps add thanos receiver as well? It also seems like an important service to monitor.

Reply:
Done!